### PR TITLE
Carousel Component

### PR DESCRIPTION
**Pull request by gingivitiss**
Basic implementation of a carousel component

### CODE REVIEW

File: src/components/BasicSection.vue [hunk 1]
```diff
@@ -3,8 +3,13 @@
         <div class="title-container">
             {{title}}
         </div>
-        <component :content-object="contentData" :is="shownComponent" />
+        <div class="component-container">
+            <component :content-object="contentData" :is="shownComponent" />
+        </div>
+        <div>
+        </div>
```
Comment (gingivitiss) [inline src/components/BasicSection.vue:10]: to delete

File: src/components/Carousel.vue [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    },
+    data() {
+        return {
+            pageCollection: [],
+            page: 0,
+            setSize: 3,
+        };
+    },
+    created() {
+        //this.groupObjects();
```
Comment (gingivitiss) [inline src/components/Carousel.vue:119]: uncomment this

**Review by stanleymok — APPROVED** [review]
pog, u added jquery. Looks good to me. Very disappointed you removed "Dis bich empty. YEET!"

**Comment by gingivitiss** [comment]
> pog, u added jquery. Looks good to me. Very disappointed you removed "Dis bich empty. YEET!"

It's still in there, I just styled it

**Pull request merged**